WooCommerce/Classes/ViewRelated/Products/Edit Product/DefaultProductFormTableViewModel.swift

```diff
@@ -199,7 +199,7 @@ private extension DefaultProductFormTableViewModel {
 
         if let stockQuantity = product.stockQuantity, product.manageStock {
             inventoryDetails.append(String.localizedStringWithFormat(Constants.stockQuantityFormat, stockQuantity))
-        } else if product.manageStock == false {
+        } else if product.manageStock == false && product.isStockStatusEnabled() {
```

Comment:
Sanity check: can this be unit tested?

Reply:
Good call! We didn't have existing unit tests, added two for simple and variable products when stock management is disabled: https://github.com/woocommerce/woocommerce-ios/pull/2786/commits/42f3c07c6deeee0ba6a81a400dedd53b5ed37ce3